ntbs-service/Pages/Alerts/Dismiss.cshtml

```diff
@@ -0,0 +1,7 @@
+@page

```

Comment:
Do we not need to specify the page route here, like:
`@page /Alerts/{AlertId:int}`?
If we use that format